.github/workflows/generate-dokka.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
     publish:
-        runs-on: macos-latest
+        runs-on: ubuntu-22.04
```

Comment:
Not sure Dokka works on `ubuntu`. I think we changed to Mac so it could generate the `darwin` docs too.